backend/prisma/schema.prisma

```diff
@@ -50,10 +31,53 @@ model User {
   username  String   @unique
   /// @z.string().email({ message: "Invalid email address" })
   email     String   @unique
-  // hash for password, set as @db.Text to remove length limit set by varchar
   hash      String   @db.Text
-  // hash for refresh token, set as @db.Text to remove length limit set by varchar
```

Comment:
pls keep comment so poeple (us in 3 months) can understand why we added the `@db.Text` part (it's a mysql specific thing)

Reply:
Reverted this change